app/views/devise/registrations/new.html.erb

```diff
@@ -14,7 +14,4 @@
   </div>
 <% end %>
 
-<%= link_to "Sign up with Facebook", user_omniauth_authorize_path(:facebook) %>
-<%= link_to "Sign up with Google", user_omniauth_authorize_path(:google_oauth2) %>
-
 <%= render "devise/shared/links" %>
```

Comment:
Adding links in devise views isn't necessary unless we want custom links such as Google or Facebook icons. 
Devise shared links already provides default links:
`<%- if devise_mapping.omniauthable? %>
  <%- resource_class.omniauth_providers.each do |provider| %>
    <%= link_to "Sign in with #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider) %><br />
  <% end -%>
<% end -%>`